src/form/dcp.jl

```diff
@@ -137,6 +137,13 @@ function constraint_voltage_angle_difference_ne(pm::GenericPowerModel{T}, n::Int
     JuMP.@constraint(pm.model, va_fr - va_to >= angmin*z + vad_min*(1-z))
 end
 
+# Should this only be for DCPLosslessForm or general DC forms?
```

Comment:
This should only apply to sub types of `DCPlosslessForm` and I think it is a good solution for that model.  We can do something better for lossy models where convex quadratic constraints are viable.